src/pages/auth/login/Login.jsx

```diff
@@ -1,10 +1,34 @@
 import React from 'react'
-
-
+import axios from 'axios'
+import {useState} from 'react'
+import { useNavigate  } from 'react-router-dom'
 import './login.css'
 import Navbar from '../../../components/navbar/Navbar'
 
 const Login = () => {
+  let navigate = useNavigate();
+  const [email , setemail ] = useState("adarshbalika@gmail.com")
+  const [password , setpassword ] = useState("adarshBalika123")
+  const [disabled , setdisabled] = useState(false)
+
+  const loginHandler = () =>{
+    setdisabled(true)
+    const bodydata = {
+      "email":email,
+      "password":password
+      
+    } 
+
+    axios.post('/api/auth/login',bodydata)
+    .then((response)=>{
+      localStorage.setItem('token',response.data.encodedToken)
+      console.log('login successfully');
+      navigate('/Listvideos')
+    },
+    (error)=>{
+      console.log('error from login page');
+    })
```

Comment:
try to set error to user cz they cant see console

Reply:
yes. will be worked on when toast is introduced. will happen by next two PRs.